tests/python/test_numpy.py

```diff
@@ -187,6 +187,13 @@ def test_index_mismatch():
         val[0, 0] = 1
 
 
+@test_utils.test()
+def test_invalid_slicing():
```

Comment:
`test_numpy.py` isn't the best place for this test :P maybe `test_field.py`?

Reply:
also maybe good to match the error message as well to make sure it fails at a place that we expected ;)

Reply:
Good point, thx!